test/1.4/methods/T_trunc.dml

```diff
@@ -24,11 +22,15 @@ method d(int64 i) -> (int5) {
     return i;
 }
 
-method test() throws {
+method init() {
     local int64 i;
-    i = a(0x5f);
+    try {
```

Comment:
similarly, remove the throws from methods that cant throw (instead of this some-hiding catch in case they somehow do throw)

Reply:
agreed